client/blocks/login/login-form.jsx

```diff
@@ -773,32 +783,312 @@ export class LoginForm extends Component {
 		this.setState( { lastUsedAuthenticationMethod: 'password' } );
 	};
 
-	render() {
+	renderLoginCard() {
+		const { lastUsedAuthenticationMethod } = this.state;
+		const isFormDisabled = this.state.isFormDisabledWhileLoading || this.props.isFormDisabled;
+		const isSubmitButtonDisabled = isFormDisabled;
+		let loginUrl;
+
 		const {
-			accountType,
 			oauth2Client,
 			requestError,
 			socialAccountIsLinking: linkingSocialUser,
-			isJetpack,
-			isJetpackWooDnaFlow,
-			currentQuery,
-			showSocialLoginFormOnly,
 			isWoo,
-			isWooJPC,
 			isBlazePro,
 			hideSignupLink,
 			isSignupExistingAccount,
 			isSendingEmail,
 			isSocialFirst,
+			isJetpack,
 		} = this.props;
 
+		const isPasswordHidden = this.isUsernameOrEmailView();
```

Comment:
Is this section moved from lines 923 of the previous version?

Reply:
Yes, most of the changes here are just restructuring of the code. I created some helper render functions like renderLoginCard, so it's easier to define our needed layout with those for Jetpack